modules/openapi-generator/src/main/resources/php-mezzio-ph/README.md.mustache

```diff
@@ -5,9 +5,9 @@ Generated by the [OpenAPI Generator](https://openapi-generator.tech) project.
 ## Overview
 This server stub aims to provide light, yet comprehensive structure for your API project using:
 
-- PHP: >=7.2
+- PHP: >=7.3
```

Comment:
Maybe it makes sense to leave a note about PHP 8 support in README? If you didn't test it carefully under 8.0 then I would add a notice about it.

Reply:
As I wrote [in previous comment](https://github.com/OpenAPITools/openapi-generator/pull/9445#issuecomment-845716159) it is tested and works against 8.0 :) I can add extra text to emphasize that but aren't  requirements in [composer.json](https://github.com/OpenAPITools/openapi-generator/blob/master/samples/server/petstore/php-mezzio-ph/composer.json) enough?